canary/requirements.txt

```diff
@@ -1,5 +1,15 @@
-python-dateutil
-viam-sdk
-pymongo
-numpy
-slack-sdk
+dnspython==2.7.0
```

Comment:
I think the better way to do this, rather than explicitly freezing including all transitive deps, is to set just the packages we directly rely on with "compatible" version markers. Given what is in this `freeze` result, I think that'd look like:
```
numpy ~= 2.2
pymongo ~= 4.11
python-dateutil ~= 2.2
slack-sdk ~= 3.34
viam-sdk ~= 0.38
```

Reply:
changed